mxcube3/routes/main.py

```diff
@@ -39,17 +39,11 @@ def unrestricted_serve_static_file():
     def get_ui_properties():
         return app.get_ui_properties()
 
-    @bp.route("/version")
```

Comment:
I don't mind if we remove this route and aggregate it with another route (is it for reducing the number of routes ?). However I'm would rather in that case that we rename mode to something with more meaning because its IMO odd to have a route called mode that also gives the server version. 

I don't have a good name for it, something like application_info or settings ?

Reply:
changed to application_settings, but unsure what would best name 🤨

I had to merge these two since they were coming out of order often, and the UI was expecting an order.

Reply:
I see, sure why not `application_settings`, we should just be careful that it does not become a dark corner for various "data" :). We could move the options that are in the `.env` file for phase control and so on as well :), not necessarily in this PR but I'm all for it of you would like to give it a go :+1:

Reply:
on a next PR ;)